frame/babe/src/default_weights.rs

```diff
@@ -47,6 +47,8 @@ impl crate::WeightInfo for () {
 			.saturating_add(110 * WEIGHT_PER_MICROS)
 			.saturating_add(25 * WEIGHT_PER_MICROS * MAX_NOMINATORS)
 			.saturating_add(DbWeight::get().reads(14 + 3 * MAX_NOMINATORS))
-			.saturating_add(DbWeight::get().writes(10 + 3 * MAX_NOMINATORS))
```

Comment:
is this file auto-generated? if so we should not edit it. 

and it is not auto-genrated, it is some weird looking code.